src/main/java/com/teragrep/lsh_01/NettyHttpServer.java

```diff
@@ -46,12 +46,9 @@ public class NettyHttpServer implements Runnable, Closeable {
     private final ServerBootstrap serverBootstrap;
     private final String host;
     private final int port;
-    private final int connectionBacklog = 128;
 
     private final EventLoopGroup processorGroup;
     private final ThreadPoolExecutor executorGroup;
-    private final HttpResponseStatus responseStatus;
-    private final InternalEndpointUrlConfig internalEndpointUrlConfig;
 
     public NettyHttpServer(
```

Comment:
We had some discussion about using primary and secondary ctors, but we need to get back to that

Reply:
So primary and secondary constructors should be used here